WordPress/src/main/java/org/wordpress/android/support/ZendeskHelper.kt

```diff
@@ -21,6 +21,7 @@ import org.wordpress.android.util.currentLocale
 import org.wordpress.android.util.AppLog
 import org.wordpress.android.util.AppLog.T
 import org.wordpress.android.util.DeviceUtils
+import org.wordpress.android.util.LanguageUtils
```

Comment:
It looks like the lint failed because of the unused `import java.util.Locale`. 

Please see [Coding Style](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/docs/coding-style.md) for more info on how to check and fix this locally.

Reply:
Fixed, thanks for the tips!